app/controllers/planning_applications/assessment/consultees_controller.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+module PlanningApplications
+  module Assessment
+    class ConsulteesController < ApplicationController
+      include CommitMatchable
+
+      before_action :set_planning_application
+      before_action :set_consultation
+
+      def index
+        respond_to do |format|
+          format.html
+        end
+      end
+
+      def check
+        @consultation.create_assessor_consultee_review
```

Comment:
I've just added a generic error message saying it can't be saved (for example, if something strange happens with the database). Because the assessor isn't passing any params, there isn't really any action they can take to remedy it - not sure if there is a better way to handle this?